app/services/pdf_service.rb

```diff
@@ -7,6 +7,10 @@ class PdfService
       left: "1cm",
       right: "1cm",
     },
+    viewport: {
+      width: 800,
```

Comment:
Has this had any effect, or am I barking up the wrong tree? If it works the way I hoped it works, changing the width from 640 to 800 wouldn't get rid of all breakpoints in the accessibility metadata, but we'd maybe see them shift along a bit (without affecting how the PDF actually looked). Then it would just be a case of finding a width that was enough to remove them entirely.